Jenkinsfile.private

```diff
@@ -0,0 +1,30 @@
+pipeline {
+	agent { node { label 'lisky' } }
+	stages {
+		stage('Prepare workspace') {
+			steps {
+				checkout scm
```

Comment:
We don't want a `deleteDir()` because of the checksum? Will this cause us problems?

Reply:
It was added a while back to workaround -- I assume -- tests not cleaning up after themselves; which is not relevant here.